src/spikeinterface/extractors/tests/test_nwbextractors_streaming.py

```diff
@@ -290,7 +290,14 @@ def test_sorting_s3_nwb_zarr(tmp_path):
 
     # test to/from dict
     sorting_loaded = load_extractor(sorting.to_dict())
-    check_sortings_equal(sorting, sorting_loaded)
+
+    # just take 3 random units to test
+    rng = np.random.default_rng(seed=2205)
```

Comment:
This seed is my brithday :)
You can put your own one if you want!

Reply:
I'll celebrate your birthday every time I do a randomised test ;)